client/components/eventform/eventform.controller.js

```diff
@@ -0,0 +1,119 @@
+
+
+'use strict';
+
+angular.module('venueApp')
+  .controller('EventFormCtrl', function($scope, Auth, EventInfo, User, SectionEvent){
+    var eventInfoId = null;
+
+    $scope.courseCreated = false;
+    $scope.selectingEvent = true;
+
+    EventInfo.getAll({}, (eventinfos) => {
+      $scope.eventinfos = eventinfos;
+    });
+
+    User.get({withSections:true, withSectionCourse:true}, (user) => {
+      var coursesObj = {};
+      user.sections.forEach((section) => {
+        var course = coursesObj[section.course._id] || section.course;
+        course.sections = course.sections || [];
+        course.sections.push(section);
+        coursesObj[section.course._id] = course;
+      });
+      $scope.courses = _.values(coursesObj);
+    });
+
+    $scope.useEvent = (event)=>{
+      eventInfoId = event._id;
+      $scope.selectingEvent = false;
+      $scope.eventInfo = event;
+    };
+
+    $scope.createEventInfo = (form)=>{
```

Comment:
Neither author nor creation date need to be added (the first is added in the controller, the second is added by mongoose serverside). The time does need to be added, but we may want a robust time selection system- so im recommending putting it in the PR with location